docusaurus/docusaurus.config.js

```diff
@@ -503,6 +503,37 @@ const config = {
           //   label: 'Blog',
           //   position: 'right'
           // },
+          {
+            type: 'dropdown',
+            label: 'Docs',
```

Comment:
the label is wrong: 'Docs'

Reply:
This can be anything like "Ziti Docs". I intended to imply the name of the site you're looking at.

Reply:
This is why i'm pointing it out. it's odd as "Docs"
![image](https://user-images.githubusercontent.com/46322585/206756635-621126c2-1f19-4f1e-b904-924b067b8638.png)

Reply:
I intended it to imply the thing you're already looking at, not the contents of the dropdown. I did this because it made sense to emulate that pattern on other sites, but I'm not opinionated about it.